applications/FemToDemApplication/custom_elements/femdem3d_element.cpp

```diff
@@ -124,7 +124,7 @@ void FemDem3DElement::InitializeInternalVariablesAfterMapping()
 
 void FemDem3DElement::ComputeEdgeNeighbours(ProcessInfo &rCurrentProcessInfo)
 {
-	std::vector<std::vector<Element *>> EdgeNeighboursContainer;
+	std::vector<std::vector<Element *>> p_edge_neighboursContainer;

```

Comment:
Technically the p is inside the vector, so I am not sure about the name @philbucher ?

Reply:
I guess I would call it `edge_neighbours_container`